src/services/documentFormat.js

```diff
@@ -35,7 +35,7 @@ export const getUpdatedSchema = (jsonDocument, collection) => {
   let properties = {}
   let schema = {}
 
-  if (type === 'object' || type === 'nested') {
+  if ((type === 'object' && jsonDocument != null) || type === 'nested') {
```

Comment:
It can't be, it comes from elasticsearch, if it is undefined the attribute won't even be there